### ISSUE TITLE
Issues with dying 

### ISSUE THREAD

**Issue by Mercurian** [EasyFarm/EasyFarm#222]
Read exactly what I say, I'm going to lay it out exactly what the issue is;

I set my account to run around an area with the wander/discover @ 20' MOST of the time it works out perfectly, but an issue always ends up happening eventually. This is that issue.

A mob will be at 20' and I'll start the attack. It will wander beyond the 20' and then aggro. I will stop attacking but be stuck locked on to the mob. ATTEMPT to attack another mob, and be unable to target the aggroed mob. Then subsequently end up dead because im locked onto one mob while trying to hit a different mob, but actually attack neither.

This is not an autotarget issue. It's an unavoidable issue that happens all the time, can you figure out a way to fix that? 

You could say "increase the detection range so that doesn't happen" if I make it larger, i'll get stuck on walls/trees/rocks. The wiggle mechanic is good for small obstacles but it doesnt work for cliffs/walls.

**Comment by Mykezero** [comment]
Is this the problem we are trying to solve?

1. Player begins attack on mob A. 
2. Mob walks out of distance, player stops attack on mob A.
3. Mob A aggros player while player engages new mob B.
4. Player now facing mob B, but is still targeting mob A.

I might be able to do something about this by adding more checks to synchronize engaging, targeting and facing the same target.

**Comment by Mercurian** [comment]
Yes, that is exactly what happens. It's pretty rare... but it's death everytime.

**Comment by Mykezero** [comment]
Try the new release: EasyFarm 1.0.89. 

I've added additional checks to all combat states to switch targets if the mob we are targeting is not the same as the mob we want to fight.

**Comment by L57703** [comment]
1.0.89 doesn't seem to work. It will target a mob, run to it, and just follow it around not attacking even if it aggros you. If i help it kill the 1st mob it will work for 1 or 2 more then it just follows stuff around again. Going back to 1.0.88 it works but same problem as Mercurian mentioned.

**Comment by schmia10** [comment]
@L57703 This is the same exact issue I am having. The most I've got is 10-15 kills before I end up following a mob around. It seems to randomly target a mob too fast after the targeted mob dies.

**Comment by Mercurian** [comment]
As of this current update, and using v1.0.91 the same issue still happens.

I usually wait until an update of ffxi to bother getting the new Easy Farm... But the targeting issue is still prevalent.

**Comment by Mykezero** [comment]
These changes might have failed because of the behavior described in issue #228
